hr_holidays_type_note/static/description/index.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8" />
+</head>
+<body><a href="http://www.gnu.org/licenses/gpl-3.0-standalone.html">
+        <img alt="License: GPL-3" src="https://img.shields.io/badge/licence-GPL--3-blue.svg" />
+    </a>
+    <section id="hr-holidays-negative-hours">
+        <h1>Hr Holidays Negative Hours</h1>
+        <p>Module summary.</p>
```

Comment:
You have to run `task lint-module` to update the `index.html`.

Reply:
@janikvonrotz the linter did not update the description, I fixed it manually.

Reply:
@laszlokr The command `task lint-module` does update the index.html otherwise you have an error in your dev env. Can you check?